.github/workflows/rust.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     - name: Fuse dependency
       run: sudo apt-get install -y libfuse-dev pkg-config
     - name: Download pak files
-      run: cd pak-examples && ./download.sh
+      run: cd pak-examples && chmod u+x download.sh && ./download.sh
```

Comment:
Oh and you can just check-in files with the execute bit set. But I guess maybe not using Windows?

Reply:
Yeah, tried that using cygwin, but did not change anything.

Reply:
Yeah, with cygwin it's still the Windows filesystem. Well, NTFS has in fact complex access control lists where execute rights could be controlled, but I'm pretty sure Windows git isn't looking at such things (especially since Windows also has FAT32, which has no permissions whatsoever). It's not the basic 9 bit Unix permissions system (and also not compatible with POSIX ACLs).